lib/puppet-syntax.rb

```diff
@@ -7,12 +7,22 @@
 module PuppetSyntax
   @exclude_paths = []
   @future_parser = false
-  @hieradata_paths = ["**/data/**/*.*yaml", "hieradata/**/*.*yaml", "hiera*.*yaml"]
+  @hieradata_paths = [
+    "**/data/**/*.*yaml",
+    "hieradata/**/*.*yaml",
+    "hiera*.*yaml"
+  ]
   @fail_on_deprecation_notices = true
   @app_management = Puppet::PUPPETVERSION.to_i >= 5 ? true : false
+  @check_hiera_keys = false
 
   class << self
-    attr_accessor :exclude_paths, :future_parser, :hieradata_paths, :fail_on_deprecation_notices, :epp_only
+    attr_accessor :exclude_paths,
```

Comment:
Same here: I moved these to new lines to make the diffs nicer, but I can do it in a separate PR if needed